src/shared/utils/price-conversion.ts

```diff
@@ -22,3 +23,44 @@ export const calculateDisplayPrice = (
     return baseCalculatedPrice / divisor;
   }
 };
+
+/**
+ * Collect the quote assets from all trading pairs.
+ */
+export const collectQuoteAssets = (summaryData: SummaryData[][]): string[] => {
+  let quoteAssetsList: string[] = [];
+  if (summaryData) {
+    summaryData.forEach(page => {
+      page.forEach(item => {
+        const quoteAsset = item.quoteAsset.symbol;
+        quoteAssetsList.push(quoteAsset);
+      });
+    });
+  }
+
+  return quoteAssetsList;
+};
+
+/**
+ * Calculates USDC-normalized amounts for trading pair values (volume or liquidity)
+ */
+export const calculateScaledAmount = (
```

Comment:
shared price conversion utility

Reply:
thought: wonder if there is something in `calculateDisplayPrice()` that relates to this. Feels like we should create a class or some type of math scaling utility that can handle all price/amount scaling between different denoms.

Reply:
+1, could be nice to extend the `pnum` to allow `add`, `multiply`, `subtract` methods